hikari/api/rest.py

```diff
@@ -4130,6 +4130,59 @@ async def edit_guild(
             If an internal error occurs on Discord while handling the request.
         """
 
+    @abc.abstractmethod
+    async def set_guild_incident_actions(
+        self,
+        guild: snowflakes.SnowflakeishOr[guilds.PartialGuild],
+        *,
+        invites_disabled_until: undefined.UndefinedNoneOr[datetime.datetime] = undefined.UNDEFINED,
+        dms_disabled_until: undefined.UndefinedNoneOr[datetime.datetime] = undefined.UNDEFINED,
+    ) -> guilds.GuildIncidents:
+        """Set the incident actions for a guild.
+
+        Parameters
+        ----------
+        guild
+            The guild to set the incident actions for. This may be the object
+            or the ID of an existing guild.
+        invites_disabled_until
+            If provided and a [`datetime.datetime`][], the datetime when invites will be enabled again. If provided and
+            [`None`][], or if not provided, invites will be enabled again immediately.
+
+            !!! note
+                If [`hikari.guilds.GuildFeature.INVITES_DISABLED`][] is active, this value will be ignored.
```

Comment:
Will it be fully ignored (ie, it will act as if it was not sent) or just not really do anything?

Reply:
After some testing, it seems like it will be fully ignored